MoltenVK/MoltenVK/GPUObjects/MVKShaderModule.mm

```diff
@@ -517,19 +482,8 @@ static uint32_t getWorkgroupDimensionSize(const SPIRVWorkgroupSizeDimension& wgD
 
 			break;
 		}
-		default:											// Could be GLSL source code
-			if (getEnabledExtensions().vk_NV_glsl_shader.enabled) {
-				const char* pGLSL = (char*)pCreateInfo->pCode;
-				size_t glslLen = codeSize - 1;
-
-				uint64_t startTime = getPerformanceTimestamp();
-				codeHash = mvkHash(pGLSL, codeSize);
-				addPerformanceInterval(getPerformanceStats().shaderCompilation.hashShaderCode, startTime);
-
-				_glslConverter.setGLSL(pGLSL, glslLen);
-			} else {
-				setConfigurationResult(reportError(VK_ERROR_INVALID_SHADER_NV, "vkCreateShaderModule(): The SPIR-V contains an invalid magic number %x.", magicNum));
-			}
+		default:
+			setConfigurationResult(reportError(VK_ERROR_INVALID_SHADER_NV, "vkCreateShaderModule(): The SPIR-V contains an invalid magic number %x.", magicNum));
```

Comment:
`VK_ERROR_INVALID_SHADER_NV` is part of `NV_glsl_shader`. 

Please revert all uses of `VK_ERROR_INVALID_SHADER_NV` to `VK_ERROR_INITIALIZATION_FAILED`.